addons/sale_timesheet/report/timesheets_analysis_views.xml

```diff
@@ -16,7 +16,55 @@
     <record id="timesheets_analysis_report_graph_inherit" model="ir.ui.view">
         <field name="name">timesheets.analysis.report.graph</field>
         <field name="model">timesheets.analysis.report</field>
-        <field name="inherit_id" ref="hr_timesheet.timesheets_analysis_report_pivot_employee"/>
```

Comment:
It is not stable since you completely change the view in the inheritance. 🙈 

you should create a new record for `hr_timesheet.timesheets_analysis_report_graph_employee` instead and add a comment to change that record and remove the new record in master. 🙂

Reply:
Yeah I agree, done with the changes. :slightly_smiling_face: